### DISCUSSION TITLE
EJBCA Community edition installation

### DISCUSSION THREAD

**Discussion by NANDOMEDIA** [Keyfactor/ejbca-ce#417] [Q&A]
Hi, I m writing from Nigeria in West Africa. Please could anyone please assist me in installing EJBCA Community version for my learning purpose. EJBCA is not known in this part of the world or used and I want to learn about it very much, but can install it on my system or server. Please can anyone help?

**Comment by aleloco09** [comment]
Hi, I am a student like you and I am getting familiarized with EJBCA environment as well without no much background knowledge of PKI.
My suggestion is to first learn the core functions of EJBCA using a containerized environment (i.e. docker) and then set up a real server.
Anyways [here](https://doc.primekey.com/ejbca/ejbca-installation) you find the documentation on how to install, deploy and run EJBCA on your host.
If you want to try with the containers [here](https://doc.primekey.com/ejbca/tutorials-and-guides) there are plenty of tutorials, and they have a [YouTube channel](https://www.youtube.com/watch?v=oWC5vsGWXQ4&list=PLt17f5skfOPFbNd8LQs-1AiPryJgcSvmZ&index=6) as well, where they clearly show you the different functions. They are setting up a playlist of video tutorials on how to use it, and it is constantly updated.
Personally I've found it really useful.

**Reply by NANDOMEDIA** [reply]
If I give you remote access to a new Server that I have, Can you help me install it?
